src/common/config.cc

```diff
@@ -36,13 +37,25 @@ static const std::string kUlDataFilePrefix =
     kExperimentFilepath + "LDPC_orig_ul_data_";
 static const std::string kDlDataFilePrefix =
     kExperimentFilepath + "LDPC_orig_dl_data_";
+static const std::string kUlPilotFreqPrefix =
+    kExperimentFilepath + "ue_pilot_data_f_";
+static const std::string kUlDataFreqPrefix = kExperimentFilepath + "ul_data_f_";
```

Comment:
This file is only used when we are testing Agora with Sounder. Maybe we have have const in `symbols.h` to only generate this file in that scenario.